.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: generic
+
+os: osx
+osx_image: xcode6.4
+
+branches:
+  only:
+  - master
```

Comment:
Added to skip needless CI builds when doing PRs against this repo. Can be dropped if this is moved to an org.